meta/meta.go

```diff
@@ -545,6 +545,12 @@ func (m *Meta) SetDDLTables() error {
 	return errors.Trace(err)
 }
 
+// SetMDLTables write a key into storage.
+func (m *Meta) SetMDLTables() error {
+	err := m.txn.Set(mDDLTableVersion, []byte("2"))
```

Comment:
it is same as lin 544 all set mDDLTableVersion, is it correct?

Reply:
Yes, the key is mDDLTableVersion, and the value is 1, 2, 3, according to the version.